e2e/cloudfront/cloudfront_suite_test.go

```diff
@@ -38,7 +38,7 @@ var _ = BeforeSuite(func() {
 	copilotCLI, err := client.NewCLI()
 	Expect(err).NotTo(HaveOccurred())
 	cli = copilotCLI
-	appName = fmt.Sprint(timeNow)
+	appName = fmt.Sprintf("t%d", timeNow)
```

Comment:
Maybe we can keep the suite name for the prefix?

Reply:
https://github.com/aws/copilot-cli/pull/4938#discussion_r1210677373 the certificate domain would have been too long

Reply:
Correction though: the limit was 64 characters I think.

`"test.t1685474080.cloudfront.copilot-e2e-tests.ecs.aws.dev"`: 57 characters
`""test.t1685474080.app-with-domain.copilot-e2e-tests.ecs.aws.dev"": 62 characters